src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -1324,15 +1359,23 @@ export const selectStakingOpportunitiesDataFull = createDeepEqualOutputSelector(
         // Delegated/Redelegated + Undelegation
         totalDelegations,
         // Rewards at 0 index: since we normalize staking data, we are guaranteed to have only one entry for the validatorId + assetId combination
-        rewards: stakingDataByValidator?.[validatorId]?.[assetId]?.rewards?.[0]?.amount ?? '0',
+        rewards: stakingDataByValidator
+          .reduce((acc, currentStakingDataByValidator) => {
+            acc = acc.plus(
+              currentStakingDataByValidator?.[validatorId]?.[assetId]?.rewards?.[0]?.amount ?? '0',
```

Comment:
Note: This isn't a user-facing change, we currently don't have anywhere in the app where aggregated rewards should be displayed, and effectively only reach code paths where we wrap a single AccountId